File: MovieHostingLinkGripper/src/main/java/com/sheryv/tools/movielinkgripper/search/TmdbEpisode.java

package com.sheryv.tools.movielinkgripper.search;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class TmdbEpisode {

    @JsonProperty("air_date")
    private String airDate;
    @JsonProperty("episode_number")
    private long episodeNumber;
    private long id;
    private String name;
    private String overview;
    @JsonProperty("vote_average")
    private long voteAverage;
    @JsonProperty("vote_count")
    private long voteCount;

}

File: WebsiteScraper/src/main/kotlin/com/sheryv/tools/websitescraper/service/videosearch/TmdbEpisode.kt
package com.sheryv.tools.websitescraper.service.videosearch

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class TmdbEpisode(
  val id: Long,
  val name: String,
  val overview: String,
  @JsonProperty("episode_number")
  val episodeNumber: Int,
  @JsonProperty("air_date")
  val airDate: String? = null,
  @JsonProperty("vote_average")
  val voteAverage: Long = 0,
  @JsonProperty("vote_count")
  val voteCount: Long = 0,
) {
}
